### ISSUE TITLE
contact / room list is not reachable through TAB key

### ISSUE THREAD

**Issue by vojtapolasek** [vector-im/element-web#25962]
### Steps to reproduce

To reproduce, you can press "TAB" key and see what elements are focused. It will skip the list of contact and move from "recently visited rooms" directly to "add" button and then to "room info" part, skipping the list.

The interesting thing is that the list becomes available after clicking on it with a mouse. However, this is very inconvenient form blind users.

It used to work in the past, it got broken maybe in may 2023 or something like that.

#### What did you expect?

That I will be able to focus the room list.

#### What happened instead?
The room list got skipped.

### Operating system

Arch Linux

### Browser information

Firefox 116

### URL for webapp

https://app.element.io

### Application version

1.11.38

### Homeserver

Synapse 1.89.0

### Will you send logs?

No

**Comment by t3chguy** [comment]
I cannot reproduce this, for me tab reaches the `People` header which is within the room list from where up/down arrows function as per https://www.w3.org/WAI/ARIA/apg/patterns/treeview/

**Comment by vojtapolasek** [comment]
For me it sometimes works, sometimes it does not. Do you reproduce it in Firefox as well?
I can always get to the item "add" (which I believe is the header of the contact list, right?), but pressing down arrow works only sometimes.

**Comment by vojtapolasek** [comment]
@t3chguy  what browser and screnreader are you using for reproducing this please? This issue still exists for me.

**Comment by t3chguy** [comment]
I used macOS + Chrome + Voiceover

**Comment by vojtapolasek** [comment]
I am using Linux and Orca with Firefox. I summon @pvagner  Do you think you could help reproduce this issue?

**Comment by vojtapolasek** [comment]
@t3chguy  how can I provide some debugging info which would help you to analyze this issue?

**Comment by t3chguy** [comment]
I don't think you can. I'm also unfortunately not currently tasked with this issue or anything related to it.

**Comment by pvagner** [comment]
I am suspecting this might be something setup in the matrix account.
What I did:
* Started new Firefox profile on linux with orca
* Navigated to https://app.element.io/ and logged in with @pvagner:matrix.org account.
* Used tab and shift+tab to navigate. I managed to find the room list which follows after the Add button in the tab order.
* Logged out.
* Logged again however this time with my other account @pvagner:pvagner.tk.
* This time I was unable to find the room list when navigating with tab and shift+tab.
* The account where I am unable to focus the room list is joined to two spaces. One of the spaces has some 30 rooms or so, the other has some 500 rooms or so and in total that account is joined to some 800 rooms.
* Clicking the room list with the mouse once will make it work for the browser session.
